ui/push-health/ClassificationGroup.jsx

```diff
@@ -72,6 +72,7 @@ class ClassificationGroup extends React.PureComponent {
       currentRepo,
     } = this.props;
     const expandIcon = detailsShowing ? faMinusSquare : faPlusSquare;
+    const expandTitle = detailsShowing ? 'Minus sign' : 'Plus sign';
```

Comment:
This text might be better as "click to expand" and "click to collapse".  That way it would work as a ``title`` and ``aria-label``.